assembl/static2/js/app/components/common/editAttachments.jsx

```diff
@@ -0,0 +1,37 @@
+// @flow
+import React from 'react';
+
+import type { Attachment } from './attachments';
+
+type EditAttachmentsProps = {
+  afterDelete: Function,
+  attachments: Array<Attachment>,
+  onDelete: Function
+};
+
+const EditAttachments = ({ attachments, onDelete, afterDelete }: EditAttachmentsProps) => {
+  return (
+    <div className="attachments">
+      {attachments.map((attachment) => {
+        const { externalUrl, title } = attachment.document;
+        return (
+          <div className="attachment" key={attachment.entityKey}>
+            <span className="assembl-icon-text-attachment" />
+            <a href={externalUrl} target="_blank" rel="noopener noreferrer">
+              {title || externalUrl}
+            </a>
+            <span
+              className="assembl-icon-delete"
+              onMouseDown={() => {
+                return onDelete(attachment.document.id);
+              }}
+              onMouseUp={afterDelete}
```

Comment:
That's because if I use `onClick` here, the `onBlur` event of the rich text editor component triggers and the attachment is not deleted. I have to use the state to know if we are deleting an attachment to do nothing in the `onBlur`
See this commit : https://github.com/assembl/assembl/commit/10d714e5746c52e1cfcfebd7f491200e50340900